docs/04-api-reference/front-commerce-core/key-values.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Key-Values"
```

Comment:
**nitpick:** there is no direct relationship with the `"@front-commerce/core/keyValues";` import name. It makes discoverability harder for a reference.

**suggestion:**  keep names intact
```suggestion
title: "keyValues"
```

Reply:
fixed